lib/resque/tasks.rb

```diff
@@ -15,6 +15,7 @@
       worker.verbose = ENV['LOGGING'] || ENV['VERBOSE']
       worker.very_verbose = ENV['VVERBOSE']
       worker.cant_fork = true if ENV['FORK_PER_JOB'] == 'false'
+      worker.graceful_term = ENV['GRACEFUL_TERM']
```

Comment:
While this isn't strictly necessary (because we don't run workers via [`resqued`](https://github.com/spraints/resqued) in all environments), it seemed worth adding for consistency with other configuration values.